core/templates/dev/head/domain/utilities/UrlInterpolationService.js

```diff
@@ -170,6 +170,13 @@ oppia.factory('UrlInterpolationService', [
         }
         return getCachePrefixedUrl('/extensions/interactions/' +
           interactionId + '/static/' + interactionId + '.png');
+      },
+
+      getComponentUrl: function(componentPath) {
+        validateResourcePath(componentPath);
+        var templatePath = ((GLOBALS.MINIFICATION || !GLOBALS.DEV_MODE) ?
+          '/templates/head/components' : '/templates/dev/head/components');
```

Comment:
Can we pass these paths from the backend, the same way things are handled in getCachePrefixedUrl()? That way, we just have one way of doing things.